tasks/deploy.yml

```diff
@@ -1,6 +1,19 @@
 ---
 # tasks file for loki deployment
 
+- name: Create Loki group
+  ansible.builtin.group:
+    name: loki
+    state: present
+
+- name: Create Loki user and add to Loki group
+  ansible.builtin.user:
+    name: loki
+    group: loki
+    createhome: false
+    system: false
+    state: present
```

Comment:
The `when` statement is missing, which should trigger the creation of users/groups only when it's required. The RPM itself, for example, manages this part.
Additionally, I noticed that you selected `system: false`, but it should actually be `true`.